File: programmers/etc/level2/최댓값과_최솟값.py
def solution(s):
    s = list(map(int, s.split()))
    return str(min(s)) + " " + str(max(s))


s = "1 2 3 4"
print(solution(s)) # "1 4"

s = "-1 -2 -3 -4"
print(solution(s)) # "-4 -1"

s = "-1 -1"
print(solution(s)) # "-1 -1"

File: Java/programmers/etc/level2/최댓값과최솟값.java
package CodeTest.Java.programmers.etc.level2;

public class 최댓값과최솟값 {
    public String solution(String s) {
        String[] answer = s.split(" ");
        int _min = Integer.MAX_VALUE, _max = Integer.MIN_VALUE;
        
        for (String a : answer) {
            int tmp = Integer.parseInt(a);
            _min = Math.min(_min, tmp);
            _max = Math.max(_max, tmp);
        }
        
        return String.valueOf(_min) + " " + String.valueOf(_max);
    }
}
